addons/mail/static/src/core/common/suggestion_service.js

```diff
@@ -150,7 +150,7 @@ export class SuggestionService {
                 .filter((persona) => persona.type === "partner");
         } else {
             partners = Object.values(this.store.Persona.records).filter(
-                (persona) => persona.type === "partner"
+                (persona) => persona != this.store.odoobot && persona.type === "partner"
```

Comment:
this "else" does not mean "chatter", it's just meant as a condition on some channel types

Reply:
Also comparing records should use `eq()`/`notEq()`/`in()`/`notIn()`/`includes()` rather than `==`.
Reference equality is not guaranteed because of how `reactive()` works, therefore it should not be used to compare reactive objects.